Sources/Swift/Integrations/UserFeedback/Configuration/SentryUserFeedbackThemeConfiguration.swift

```diff
@@ -88,24 +88,18 @@ public class SentryUserFeedbackThemeConfiguration: NSObject {
      */
     public var buttonBackground: UIColor = UIColor.clear
     
-    /**
-     * Color used for success-related components (such as text color when feedback is submitted successfully).
-     * - note: Default light mode: `rgb(38, 141, 117)`; dark mode: `rgb(45, 169, 140)`
-     */
-    public var successColor = UIScreen.main.traitCollection.userInterfaceStyle == .dark ? UIColor(red: 45 / 255, green: 169 / 255, blue: 140 / 255, alpha: 1) : UIColor(red: 38 / 255, green: 141 / 255, blue: 117 / 255, alpha: 1)
```

Comment:
another thing that we weren't currently using for any kind of default success UX